### ISSUE TITLE
[Poll] Deprecate daily directory

### ISSUE THREAD

**Issue by silascutler** [Malshare/MalShare#38]
A list of hashes each day are saved to `malshare.com/daily/`, however, this same data is also available via API.  These digest files are generated by an external script managed by a cron job.  

Would it be worth deprecating this?

**Comment by silascutler** [comment]
@larsborn / @adrenaline96  - Thoughts?

**Comment by larsborn** [comment]
We could just do a 302 redirect to the corresponding API endpoint that delivers the same information that was present in the files.

**Comment by adrenaline96** [comment]
Yeah @larsborn is correct, the redirect is the best approach. malshare.com/daily is not worth keeping in its current form (in my opinion), so the redirect is the best thing.

**Comment by adrenaline96** [comment]
Also @silascutler I see you made some changes to the API, time to update the .NET implementation as well, working on it.

**Comment by larsborn** [comment]
We are now generating those automatically with a Python script.
